concourse/pipelines/create-microbosh.yml

```diff
@@ -120,11 +122,14 @@ jobs:
           - -e
           - -c
           - |
-            ./paas-cf/manifests/bosh-manifest/scripts/generate-bosh-secrets.sh > bosh-secrets.yml
-            ls -l bosh-secrets.yml
+            ./paas-cf/manifests/bosh-manifest/scripts/generate-bosh-secrets.sh > generate/bosh-secrets.yml
+            ls -l generate/bosh-secrets.yml
+        outputs:
+        - name: generate
```

Comment:
We have 2 patters how we do outputs, I went with the pattern that's used locally. Meaning in bosh pipelines it's inputs, script, outputs; in CF it's in, out, script. We should pick one pattern and refactor all pipelines to use it. Likely when merging. I wanted to do only minimal changes for this PR - and this is also why the outputs are named same as task, as this is how they were imported previously. If I change this name, I would have to also change all the tasks that import it. The story here is to update concourse and not to refactor pipeline. But I think this is also one of the things we should fix.